### PR TITLE
Remove gist links to my personal GitHub account

### PR DESCRIPTION
**Pull request by droberts-ada**
## Description
Exactly what it says on the tin. I did double check that all links go to the right place.

There are a number of gist links still in the Backbone section, but that's deprecated anyway so I figured it's not worth the trouble.

## Questions for the Author
- [ ] N/A Have you updated the weekly learning goals in the pedagogy resource?
- [ ] N/A Have you updated the weekly assessment in Schoology?
- [ ] N/A Have you added links to the source (lucidchart, draw.io) for any diagram?
- [ ] N/A Have you updated the READMEs with links to new or moved resources?

## Todos
None

## Feedback Requested
Nothing special

### CODE REVIEW

File: 02-intermediate-ruby/testing-with-minitest.md [hunk 1]
```diff
@@ -259,7 +259,15 @@ You may remember from the [Introduction to Automated Testing](../00-programming-
 1. The seconds are greater than or equal to 60, should raise an error
 1. Any parameter is less than 0, should raise an error
 
-**Question** Are there other edge-cases we should test for?  Do any other inputs cause different behaviors in the method?  Talk about it with your SeatSquad members.  If you are unsure check [here](https://gist.github.com/ada-instructor-1/27c2cb78b177a6139241860a6bc54712).
+<details>
+<summary>
+<b>Question:</b> Are there other edge-cases we should test for?  Do any other inputs cause different behaviors in the method?  Talk about it with your SeatSquad members, and click here for an answer if you are still unsure.
```
Comment (CheezItMan) [inline 02-intermediate-ruby/testing-with-minitest.md]: I thought we'd agreed to remove `SeatSquad` from the textbook curriculum.  I actually don't mind it, just looking for consistency.

Comment (dHelmgren) [inline 02-intermediate-ruby/testing-with-minitest.md]:
```suggestion
<b>Question:</b> Are there other edge-cases we should test for?  Do any other inputs cause different behaviors in the method?  Talk about it with your neighbors, and click here for an answer if you are still unsure.
```

Comment (droberts-ada) [reply 02-intermediate-ruby/testing-with-minitest.md]: I think that's true, but this PR is not the place for that work.
